README.md

```diff
@@ -35,13 +35,17 @@ gutil.template('test <%= name %> <%= file.path %>', opt) // test todd /js/hi.js
 
 ### log(msg...)
 
-Logs stuff. Already prefixed with [gulp] and all that. Use the right colors for values. If you pass in multiple arguments it will join them by a space.
+Logs stuff. Already prefixed with [gulp] and all that. If you pass in multiple arguments it will join them by a space.
 
 ```
 values (files, module names, etc.) = magenta
 numbers (times, counts, etc) = cyan
 ```
 
+### colors
+
+Is an instance of [chalk](https://github.com/sindresorhus/chalk).
```

Comment:
No, we'll be very soon deprecating it. Rather you say something like further up in the readme:

```
Use [`chalk`](https://github.com/sindresorhus/chalk) for colorizing console output.
```